espnet/nets/pytorch_backend/rnn/decoders_transducer.py

```diff
@@ -35,15 +35,19 @@ def __init__(self, eprojs, odim, dtype, dlayers, dunits, blank,
         self.dropout_embed = torch.nn.Dropout(p=dropout_embed)
 
         if dtype == "lstm":
-            dec_net = torch.nn.LSTMCell
+            dec_net = torch.nn.LSTM
         else:
-            dec_net = torch.nn.GRUCell
+            dec_net = torch.nn.GRU
 
-        self.decoder = torch.nn.ModuleList([dec_net(embed_dim, dunits)])
+        self.decoder = torch.nn.ModuleList([dec_net(embed_dim, dunits, 1,
```

Comment:
I believe `self.decoder = dec_net(embed_dim, dunits, dlayers, dropout=dropout, ...)` is faster than stacking each LSTM/Dropout layers because the layers are fused in cudnn implementation.

Reply:
However, if you think it is too much for this PR, you can skip this comment.

Reply:
If it's okay for you I'll compare both and change in another PR if it's faster!